docs/rules/html-self-closing.md

```diff
@@ -0,0 +1,82 @@
+---
+pageClass: "rule-details"
+sidebarDepth: 0
+title: "svelte/html-self-closing"
+description: "Enforce self-closing style"
+since: "v2.3.0"
+---
+
+# svelte/html-self-closing
+
+> Enforce self-closing style
+
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+You can choose either two styles for elements without content
+
+- always: `<div />`
+- never: `<div></div>`
+
+<ESLintCodeBlock fix>
+
+<!-- prettier-ignore-start -->
+<!--eslint-skip-->
+
+```svelte
+<script>
+  /* eslint svelte/html-self-closing: "error" */
+</script>
+
+<!-- ✓ GOOD -->
+<div />
+<p>Hello</p>
+<div><div /></div>
+<img>
+
+<!-- ✗ BAD -->
+<div></div>
+<p> </p>
+<div><div></div></div>
+<img />
+```
+
+<!-- prettier-ignore-end -->
+
+</ESLintCodeBlock>
+
+## :wrench: Options
+
+```json
+{
+  "svelte/html-self-closing": [
+    "error",
+    {
+      "html": {
+        "void": "never", // or "always" or "any"
+        "normal": "always", // or "never" or "any"
+        "custom": "always" // or "never" or "any"
+      }
+    }
+  ]
+}
+```
+
+- `html.void` (`"never"` by default)... Style of HTML void elements
```

Comment:
Could you make `"always"` the default to be the same as the default for prettier-plugin-svelte?